content/en/tracing/advanced/runtime_metrics.md

```diff
@@ -68,10 +68,26 @@ Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
 {{% /tab %}}
 {{% tab "Node.js" %}}
 
-Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
+<div class="alert alert-info">
+This feature is currently in <strong>BETA</strong>.
+Reach out to <a href="/help">the Datadog support team</a> to be part of the beta.
+</div>
 
+Runtime metrics collection can be enabled with one configuration parameter in the tracing client:
 
-[1]: /help
+* Tracer Option: `tracer.init({ runtimeMetrics: true })`
+* Environment Variable: `DD_RUNTIME_METRICS_ENABLED=true`
+
+Runtime metrics can be viewed in correlation with your Node services. See the [Service page][1] in Datadog.
+
+**Collecting Runtime Metrics in Containerized Environments**
+
+By default, Runtime metrics from your application are sent to the Datadog Agent over port 8125. If you are running the Agent as a container, ensure that `DD_DOGSTATSD_NON_LOCAL_TRAFFIC` [is set to true][2], and that port 8125 is open on the Agent. For example: in Kubernetes, [bind the DogstatsD port to a host port][3]; in ECS, [set the appropriate flags in your task definition][4].
```

Comment:
Is the port the key part, or that we use dogstatsd?

Reply:
We use DogStatsD